### PR TITLE
fix(client): fix breakies and stuff

### PR DESCRIPTION
**Pull request by mockdot**
## Description

Who needs breaks anyways

Proper Description
- Error saying vehicle = 0 on console no longer shows
- Vehicle breaks now work
- Max vehicle speed works
- Looks like as the error says, the whole handling file was not being loaded and then breaking all of its features.

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

File: client.lua [hunk 1]
```diff
@@ -475,7 +475,7 @@ CreateThread(function()
 		Wait(50)
 		local ped = cache.ped
 		if isPedDrivingAVehicle() then
-			vehicle = GetVehiclePedIsIn(ped, false)
+			vehicle = GetVehiclePedIsIn(ped, true)
```
Comment (Manason) [inline client.lua]: Why this change?

**Review by Manason — CHANGES_REQUESTED** [review]

File: client.lua (hunk 1 above)
Comment (mockdot) [reply client.lua]: Error happens on exiting a vehicle:

[ 10295266] [b2802_GTAProce] MainThrd/ no entity for vehicle 0 in GET_VEHICLE_HANDLING_FLOAT
[ 10295266] [b2802_GTAProce] MainThrd/ no script guid for vehicle 0 in SET_VEHICLE_HANDLING_FIELD

It happens after you get in and out, it looks like that variable is getting replaced by 0 when GetVehiclePedIsIn is false instead of giving the last vehicle in, so when the condition is not met below, it breaks the handling. You can test and see that if you exit a vehicle multiple times now it won't give this error any longer and the breaks will continue to work, I am guessing also the collision and damage multiplier were also being affected by this.

Comment (DevX32) [reply client.lua]: i tried it from my side it is ok no error pop ups and vehicle speed decrease before using 210 kmph after using 180kmph and vehicle breaks sucks before using that i think it fix that too ( cars used for test ( neon, t20, sultan terzo and bikes , sanchez, manchez, carbon and bati)

Comment (Manason) [reply client.lua]:
```suggestion
			vehicle = cache.vehicle
```

**Review by Manason — CHANGES_REQUESTED** [review]

**Review by Manason — APPROVED** [review]
